src/main/java/seedu/hms/model/booking/Booking.java

```diff
@@ -69,10 +69,36 @@ public int numOfCustomers() {
 
     @Override
     public String toString() {
+        String otherUsersToString = "";
+        if (otherUsers.isPresent()) {
+            for (Customer u : otherUsers.get()) {
+                otherUsersToString += u.getName();
+            }
+        }
         return "Booking for "
             + service.getName()
             + " from " + timing
             + " by " + payer.getName()
+            + " with " + otherUsersToString
             + ". Comment - " + comment.orElse("N/A");
     }
+
+    @Override
+    public boolean equals(Object other) {
+        if (other == this) {
+            return true;
+        }
+
+        if (!(other instanceof Booking)) {
+            return false;
+        }
+
+        Booking ob = (Booking) other;
+        System.out.println(getOtherUsers());
```

Comment:
Can remove print statements ?

Reply:
Didn't check all files - but just remove from all

Reply:
I removed all in the new commits, thanks for the review